src/components/Common/Avatar.tsx

```diff
@@ -66,15 +66,15 @@ const Avatar = React.forwardRef<
         <AvatarPrimitive.Image
           src={imageUrl}
           alt={name}
-          className="aspect-square h-full w-full object-cover"
+          className="aspect-square object-cover"
```

Comment:
it should take the full height and width of the parent element right? (i.e. AvatarPrimitive)